web-ui/src/main/resources/catalog/templates/admin/usergroup/users.html

```diff
@@ -493,9 +493,9 @@ <h3 data-translate="">UserAdmin</h3>
             <i class="fa fa-search" />&nbsp;
             <ng-pluralize
               count="searchResults.count"
-              when="{'0': ('noRecordFound' | translate),
-                    'one': '1 ' +  ('userRecord' | translate) + ' {{userSelected.name}} {{userSelected.surname}}',
-                    'other': '{} ' +  ('userRecords' | translate) + ' {{userSelected.name}} {{userSelected.surname}}'}"
+              when="{'0': {{'noRecordFound' | translate}},
+                    'one': '1 ' +  {{'userRecord' | translate}} + ' {{userSelected.name}} {{userSelected.surname}}',
+                    'other': '{} ' +  {{'userRecords' | translate}} + ' {{userSelected.name}} {{userSelected.surname}}'}"
```

Comment:
```suggestion
              when="{'0': '{{&quot;noRecordFound&quot; | translate}}',
                    'one': '1 ' +  '{{&quot;userRecord&quot; | translate}}' + ' {{userSelected.name}} {{userSelected.surname}}',
                    'other': '{} ' +  '{{&quot;userRecords&quot; | translate}}' + ' {{userSelected.name}} {{userSelected.surname}}'}"
```

The expression provided doesn't work, please @archaeogeek try the suggested change